Added model for gettext variants. (#15513)
github/codeql: cpp/ql/src/change-notes/2024-02-05-gettext-dataflows.md

---
category: minorAnalysis
---

> * Added models for gettext function variants to support dataflow through these functions. 

Reviewer: ```suggestion
* Added dataflow models for the `gettext` function variants. 
```
